backend/src/nodes/impl/color_transfer/mean_std.py

```diff
@@ -30,7 +30,7 @@ def image_stats(img: np.ndarray):
     """Get means and standard deviations of channels"""
 
     # Compute the mean and standard deviation of each channel
-    channel_a, channel_b, channel_c = cv2.split(img)
+    channel_a, channel_b, channel_c = np.split(img, 3, 1)
```

Comment:
Why?

Reply:
If I recall correctly, `cv2.split` errored here because I already reshaped `img` into a 1D image (i.e. a list of pixels), and it expected a 2D image (i.e. a matrix of pixels). If there's some other way you think I should handle that, I'm open to suggestions.

Reply:
Why is it 1D in the first place? If what you said is correct (I haven't checked and don't know `cv2.split` by heart), then it wasn't 1D before.

Reply:
I can't remember for sure why I did that (due to this PR being a few months old by now) but I think it was because filtering out the transparent pixels was more straightforward when in 1D format (I think because different lines will have different numbers of pixels filtered out).

Reply:
Pretty sure this would be a result of changing `image_stats(img)` to `image_stats(img[valid_indices])`. @RunDevelopment do you want me to add a comment clarifying this?